.env

```diff
@@ -8,5 +8,5 @@ REACT_APP_GDC_AUTH=/auth/
 REACT_APP_COMMIT_HASH=
 REACT_APP_COMMIT_TAG=
 DANGEROUSLY_DISABLE_HOST_CHECK=true
-REACT_APP_ALLOW_FAKE_USER=
+REACT_APP_ALLOW_DEV_USER=
```

Comment:
this is just a rename and it's dev-only so no need to worry, but the app's functionality allows for either name (i.e. legacy compatible)